src/main/java/core/basesyntax/BallsCollors.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum BallsCollors {

```

Comment:
The enum name 'BallsCollors' seems to be a typo. According to the Java naming conventions, enum names should be in camel case and spelled correctly. It should probably be 'BallColors'.